standards/core/peer-exchange.md

```diff
@@ -66,26 +66,40 @@ message PeerInfo {
   bytes enr = 1;
 }
 
-message PeerExchangeQuery {
+message PeerExchangeRequest {
   uint64 num_peers = 1;
 }
 
 message PeerExchangeResponse {
   repeated PeerInfo peer_infos = 1;
+  uint32 status = 10;
+  optional string desc = 11;
 }
 
 message PeerExchangeRPC {
-  PeerExchangeQuery query = 1;
+  PeerExchangeRequest request = 1;
   PeerExchangeResponse response = 2;
 }
 
 ```
 
 The `enr` field contains a Waku ENR as specified in [WAKU2-ENR](./enr.md).
 
-Requesters send a `PeerExchangeQuery` to a peer.
+Requesters send a `PeerExchangeRequest` to a peer.
 Responders SHOULD include a maximum of `num_peers` `PeerInfo` instances into a response.
 Responders send a `PeerExchangeResponse` to requesters containing a list of `PeerInfo` instances, which in turn hold an ENR.
+Responders, MUST include a `status` in the response in any case and possible descriptive form of the problem if any.
+
+### Possible status codes
+
+| Result | Code | Note |
+|--------|------|------|
+| SUCCESS  | 200 | Successfull request-respond. In response the answer must contain `PeerExchangeResponse`  |
+| BAD_REQUEST | 400   | Wrong request payload. It must only contain `reques` field.  |
```

Comment:
```suggestion
| BAD_REQUEST | 400   | Wrong request payload. It must only contain `request` field.  |
```
Again, here and elsewhere, not sure I understand the field directives here "It must only contain a `request`..." -> this is a field inside the `response`, so the existence of the response code violates its description unless I'm missing something. 😅